systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesST.java

```diff
@@ -0,0 +1,93 @@
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.Environment;
+import io.strimzi.systemtest.annotations.ParallelNamespaceTest;
+import io.strimzi.systemtest.resources.kubernetes.DeploymentResource;
+import io.strimzi.systemtest.resources.operator.BundleResource;
+import io.strimzi.systemtest.templates.crd.KafkaTemplates;
+import io.strimzi.systemtest.utils.kubeUtils.objects.PodUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Feature Gates should give us additional options on
+ * how to control and mature different behaviors in the operators.
+ * https://github.com/strimzi/proposals/blob/main/022-feature-gates.md
+ */
+public class FeatureGatesST extends AbstractST {
+
+    static final String NAMESPACE = "feature-gates-tests";
+    private static final Logger LOGGER = LogManager.getLogger(FeatureGatesST.class);
+
+    private Deployment clusterOperator;
+    private List<EnvVar> defaultEnvVars;
+    private List<EnvVar> testEnvVars;
+
+    @BeforeAll
+    void setup(ExtensionContext extensionContext) {
+        installClusterWideClusterOperator(extensionContext, NAMESPACE, Constants.CO_OPERATION_TIMEOUT_DEFAULT, Constants.RECONCILIATION_INTERVAL);
+        clusterOperator = DeploymentResource.getDeploymentFromYaml(BundleResource.PATH_TO_CO_CONFIG);
+        defaultEnvVars = clusterOperator.getSpec().getTemplate().getSpec().getContainers().get(0).getEnv();
+    }
+
+    @AfterEach
+    void afterEach(ExtensionContext extensionContext) {
+        clusterOperator = redeployCOWithEnvVars(defaultEnvVars);
+        testEnvVars = null;
+    }
+
+    @BeforeEach
+    void beforeEachSetClusterOperator(ExtensionContext extensionContext) {
+        clusterOperator = DeploymentResource.getDeploymentFromYaml(BundleResource.PATH_TO_CO_CONFIG);
+        testEnvVars = clusterOperator.getSpec().getTemplate().getSpec().getContainers().get(0).getEnv();
+    }
+
+    protected Deployment redeployCOWithEnvVars(List<EnvVar> envVarList) {
+        // Apply env variables
+        clusterOperator.getSpec().getTemplate().getSpec().getContainers().get(0).setEnv(envVarList);
+        return new DeploymentBuilder(clusterOperator).build();
+    }
+
+    /**
+     * Control Plane Listener
+     * https://github.com/strimzi/proposals/blob/main/025-control-plain-listener.md
+     */
+    @Test
+    @ParallelNamespaceTest
```

Comment:
`@IsolatedTest` here will be much better. Imagine running for example 4 tests in parallel, which will change something in CO. 
`@ParallelNamespaceTest` is for STs which will have same beginning, like the CO, and we want to deploy multiple Kafka clusters.